templates/example_cell_clustering.ipynb

```diff
@@ -17,7 +17,9 @@
   {
```

Comment:
<p>In the bullet point for cell_table_name, I think we should specify that the only columns from the cell table that are needed are fov, label, and cell_size (and that's why it doesn't matter if we use normalized or arcsinh versions, since we don't actually use the marker values in the table). Also helps people who might be using a cell table generated differently to understand what input they need here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>